### DISCUSSION TITLE
How long to expect a ticket response for GitHub for Education?

### DISCUSSION THREAD

**Discussion by jbonoUMD** [community/community#45596] [GitHub Education]
Hi,

My students can't sign up for GitHub for Education because the email domains GitHub has for our school are outdated. How long does it take for a ticket to be responded to for this? I submitted a ticket two days ago with no response yet, but need to have this resolved before Monday or I will have student issues.

**Comment by jge162** [comment]
Hello @jbonoUMD in the past it was around 24/48 hours. But, no way of knowing your exact wait time here. this is just a discussion forum. Just keep checking your emails and spam folders. or log back into education to see if your status has changed or remains the same.

**Reply by jge162** [reply]
Also if you want to get approved you should update your profile. use your real name and etc...

**Reply by jbonoUMD** [reply]
Yes, it's been longer than 48 hours at this point. I just checked the ticket and there is no response. I put info on the ticket on how to verify me by linking to my faculty page.

**Reply by jge162** [reply]
Hm I think verification is email and supporting documentation. But wait for results of your ticket. Hopefully they approve

**Reply by jbonoUMD** [reply]
I'm already approved. It is my students who cannot get approved because GitHub has the wrong domains for our school. Thanks anyway. Hopefully someone at GitHub will provide adequate support and fix the issue on their backend.

**Reply by jge162** [reply]
AH I see, you should be fine then. just the wait is annoying.
